pkg/tsdb/mssql/mssql.go

```diff
@@ -52,12 +52,14 @@ func generateConnectionString(datasource *models.DataSource) string {
 	}
 
 	server, port := hostParts[0], hostParts[1]
-	return fmt.Sprintf("server=%s;port=%s;database=%s;user id=%s;password=%s;",
+	encrypt := datasource.JsonData.Get("sslmode").MustString("false")
```

Comment:
On second thought I think we should rename the `sslmode` to `encrypt` so that it's not mixed up with the `sslmode` setting for postgres. Also because we've named it Encrypt in the UI.